resources/eventing/charts/controller/templates/webhook.yaml

```diff
@@ -0,0 +1,56 @@
+{{ if .Values.enableNewCRDVersion }}
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  creationTimestamp: null
+  name: subscription-mutating-webhook-configuration
+webhooks:
+  - admissionReviewVersions:
+      - v1
+    clientConfig:
+      service:
+        name: {{ include "controller.fullname" . }}-webhook-service
+        namespace: {{ .Release.Namespace }}
+        path: /mutate-eventing-kyma-project-io-v1alpha2-subscription
+    failurePolicy: Fail
+    name: msubscription.kb.io
+    rules:
+      - apiGroups:
+          - eventing.kyma-project.io
+        apiVersions:
+          - v1alpha2
+        operations:
+          - CREATE
+          - UPDATE
+        resources:
+          - subscriptions
+    sideEffects: None
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  creationTimestamp: null
+  name: subscription-validating-webhook-configuration
```

Comment:
(nit) is it not better to make it configurable as it is used in the code too

Reply:
done